packages/list/src/mwc-list-item-base.ts

```diff
@@ -154,9 +154,15 @@ export class ListItemBase extends LitElement {
       ];
 
   get text() {
-    const textContent = this.textContent;
-
-    return textContent ? textContent.trim() : '';
+    const primaryTextSlot = this.renderRoot.querySelector('.mdc-list-item__text')?.querySelector('slot');
+    if (!primaryTextSlot) {
+      return '';
+    }
+    return primaryTextSlot.assignedNodes().reduce(
```

Comment:
For SSR reasons, we cannot use `assignedNodes()`; please find a way to do this with `@material/mwc-base/utils.ts`'s `findAssignedElement` function